tests/testthat/test-demean.R

```diff
@@ -62,3 +62,125 @@ test_that("demean shows message if some vars don't exist", {
     regexp = "not found"
   )
 })
+
+
+# see issue #520
+test_that("demean for cross-classified designs (by > 1)", {
+  skip_if(getRversion() < "4.1.0") # for pipe
```

Comment:
Not a big fan of this. Can you rather use the pipe from `poorman` and skip if `poorman` is not installed?

Reply:
Ok, but what's the reason for not using native pipe? Even the tidyverse team suggests using it instead of magrittr's pipe.

Reply:
With the native pipe we skip R 3.6 and 4.0. Even if it's unlikely there was a change that makes this function fails for those versions I'd rather test them too